profile_bluesky/startup/instrument/devices/aps_source.py

```diff
@@ -66,4 +66,6 @@ class MyDualUndulator(Device):
 # TODO: Not sure where to put the tracking.....
 
 undulator = MyDualUndulator("ID04", name="undulator")
+undulator.tracking = False
+undulator.stage_sigs["tracking"] = True
```

Comment:
With `stage_sigs`, tracking would be set True every time a plan uses this undulator.  If you don't want that, remove the `stage_sigs` line and `bs.mv(undulator.tracking, True)` or `False` when desired.